### PR TITLE
chore(docs): Add "how to" README sections to public-facing Driver Adapters

### PR DESCRIPTION
**Pull request by jkomyno**
Closes https://github.com/prisma/team-orm/issues/387.

This PR adds instructions about how to install and use the following Driver Adapters:
- [x] PlanetScale
- [x] Neon
- [x] libSQL (Turso)

### CODE REVIEW

**Review by SevInf — APPROVED** [review]

File: query-engine/driver-adapters/js/adapter-libsql/README.md [hunk 1]
```diff
@@ -3,3 +3,93 @@
...
+
+Make sure your Turso database connection string and authentication token is copied over to your `.env` file. The connection string will start with `libsql://`.
+
+```env
+# .env
+TURSO_AUTH_TOKEN="eyJhbGciOiJFZERTQSIsInR5cCI6IkpXVCJ9..."
+TURSO_DATABASE_URL="libsql://turso-prisma-random-user.turso.io"
+```
+
+You can now reference this environment variable in your `schema.prisma` datasource. Make sure you also include the `driverAdapters` Preview feature.
```
Comment (miguelff) [inline query-engine/driver-adapters/js/adapter-libsql/README.md:26]: I don't see any env var referenced in the datasource block of the schema.prisma, but instead a local URL.

File: query-engine/driver-adapters/js/adapter-neon/README.md [hunk 2]
```diff
@@ -2,4 +2,70 @@
...
+```
+
+Make sure your [Neon database connection string](https://neon.tech/docs/connect/connect-from-any-app) is copied over to your `.env` file. The connection string will start with `postgres://`.
+
+```env
+# .env
+DATABASE_URL="postgres://..."
+```
+
+You can now reference this environment variable in your `schema.prisma` datasource. Make sure you also include the `driverAdapters` Preview feature.
```
Comment (miguelff) [inline query-engine/driver-adapters/js/adapter-neon/README.md:26]: I see, maybe in the case of turso, the text was wrong due to copypasta?

File: query-engine/driver-adapters/js/adapter-libsql/README.md (hunk 1 above)
Comment (jkomyno) [reply query-engine/driver-adapters/js/adapter-libsql/README.md:26]: Thank you, this is a copy-paste error of me trying to fit [Turso snippets](https://www.prisma.io/blog/prisma-turso-ea-support-rXGd_Tmy3UXX) with the same structure used in [PlanetScale and Neon's blog post](https://www.prisma.io/blog/serverless-database-drivers-KML1ehXORxZV).

File: query-engine/driver-adapters/js/adapter-neon/README.md (hunk 2 above)
Comment (jkomyno) [reply query-engine/driver-adapters/js/adapter-neon/README.md:26]: 🍝

File: query-engine/driver-adapters/js/adapter-libsql/README.md [hunk 3]
```diff
@@ -3,3 +3,93 @@
...
+4. Pass the driver adapter to the Prisma Client instance
+
+```typescript
+// Import needed packages
+import { PrismaClient } from '@prisma/client';
+import { PrismaLibSQL } from '@prisma/adapter-libsql';
+import { createClient } from '@libsql/client';
+
+// Setup
+const connectionString = `${process.env.TURSO_DATABASE_URL}`;
```
[the remark this answers is not in the document]
Comment (jkomyno) [reply query-engine/driver-adapters/js/adapter-libsql/README.md:77]: I believe the clarification feedback should also be made on https://www.prisma.io/blog/prisma-turso-ea-support-rXGd_Tmy3UXX.

**Review by miguelff — APPROVED** [review]

File: query-engine/driver-adapters/js/adapter-libsql/README.md [hunk 4]
```diff
@@ -3,3 +3,93 @@
...
+  provider = "sqlite"
+  url      = "file:./dev.db"
+}
+```
+
+Now run `npx prisma generate` to re-generate Prisma Client.
+
+## How to setup migrations
+
+As Turso needs to sync between a local sqlite database and another one hosted on Turso Cloud, an additional migration setup is needed. In particular, anytime you modify models and relations in your `schema.prisma` file, you should:
```
Comment (janpio) [inline query-engine/driver-adapters/js/adapter-libsql/README.md:45]: The first sentence here is just wrong. There is no local SQLite database involved when using Prisma. There might be weird things needed for our migration flow as we do not support Migrations with Turso yet.

**Pull request merged**